app/views/accounts/_pii.html.slim

```diff
@@ -1,9 +1,13 @@
-.mb3.profile-info-box
-  - if locked_for_session
+- if locked_for_session
+  .mb2
     .usa-alert.usa-alert--warning
       .usa-alert__body
         .usa-alert__text
           = t('account.re_verify.banner')
+          '
+          = link_to(t('account.re_verify.footer'), '/reauthn')
```

Comment:
I know the path didn't change here, it just moved, but I think we should change `'/reauthn'` to: `user_password_confirm_path` -- this uses the rails router method so that if the path changes rendering the template would error